src/features/embalm/stepContent/hooks/useSarcophagusParameters.ts

```diff
@@ -90,7 +95,9 @@ export const useSarcophagusParameters = () => {
       Step.RequiredArchaeologists
     ];
 
-    return requiredSteps.every(step => getStatus(step) === StepStatus.Complete);
+    return requiredSteps
+      .filter(s => !isHardhatNetwork ? true : s !== Step.FundBundlr) // Not checking fund bundlr step when testing in hardhat
+      .every(step => getStatus(step) === StepStatus.Complete);
```

Comment:
an alternate way, just in case we wanted to target other steps conditionally in the future  without having to update the `requiredSteps` filter below (not that we ever will) -

```
const requiredSteps = [
      Step.NameSarcophagus,
      Step.UploadPayload,
      isHardhatNetwork ? undefined : Step.FundBundlr,
      Step.SetRecipient,
      Step.SelectArchaeologists,
      Step.RequiredArchaeologists
    ].filter(step => step !== undefined)
```

Reply:
Noice